pkg/agent/flowexporter/connections/connections.go

```diff
@@ -0,0 +1,120 @@
+package connections
+
+import (
+	"strconv"
+	"sync"
+	"time"
+
+	"k8s.io/klog"
+
+	"github.com/vmware-tanzu/antrea/pkg/agent/flowexporter"
+	"github.com/vmware-tanzu/antrea/pkg/agent/interfacestore"
+	"github.com/vmware-tanzu/antrea/pkg/agent/openflow"
+)
+
+var _ CxnStoreAndBuilder = new(cxnStoreAndBuilder)
+
+type CxnStoreAndBuilder interface {
+	Run(stopCh <-chan struct{})
+}
+
+// TODO: Name dilemma--cxnStoreAndBuilder or cxnCache or something else?
+type cxnStoreAndBuilder struct {
+	cxns            map[[5]string]flowexporter.Connection // Add 5-tuple as string array
```

Comment:
can we defined a separate type for the connection identifier, e.g. maybe have `type connectionKey [5]string`?